doc/manual/index.rst

```diff
@@ -127,7 +126,6 @@ Additional information about various MNE-C tools.
    c_reference
    gui/analyze
    gui/browse
-   c_cookbook
```

Comment:
There is no `c_cookbook.rst` file. Maybe you're thinking of the `cookbook.rst` file, which still exists and is still linked to?